lib/main_containers.pm

```diff
@@ -228,9 +228,11 @@ sub load_container_tests {
         $run_args->{runtime} = $_;
         if (is_container_image_test()) {
             if (get_var('BCI_TESTS')) {
-                loadtest('containers/bci_test', run_args => $run_args, name => 'bci_test_' . $run_args->{runtime});
-                # For Base image we also run traditional image.pm test
-                load_image_test($run_args) if (is_sle(">=15-SP3") && check_var('BCI_TEST_ENVS', 'base'));
+                unless (get_var('BCI_SKIP')) {
```

Comment:
Note: Needs to be within the `if` block, otherwise the routine might fall in one of the other `elsif`s.